application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java

```diff
@@ -131,6 +138,71 @@ private RestAction<Message> sendExplanationMessage(GuildMessageChannel threadCha
+            .flatMap(message -> sendChatGptAttempt(message.getContentRaw(), threadChannel));
+    }
+
+    /**
+     * Determine between the title of the thread and the first message which to send to the AI. It
+     * uses a simple heuristic of length to determine if enough context exists in a question. If the
+     * title is used, it must also include a question mark since the title is often used more as an
+     * indicator of topic versus a question.
+     * 
+     * @param questionFirstMessage - The first message of the thread which originates from the
+     *        question asker.
+     * @param threadChannel - The thread in which the question was asked.
+     * @return An answer for the user from the AI service or a message indicating either an error or
+     *         why the message wasn't used.
+     */
+    private RestAction<Message> sendChatGptAttempt(String questionFirstMessage,
+            ThreadChannel threadChannel) {
+        String questionTitle = threadChannel.getName();
+        StringBuilder stringBuilder = new StringBuilder(questionFirstMessage);
+        List<ForumTag> tags = threadChannel.getAppliedTags();
+
+        if (questionFirstMessage.length() > MAX_QUESTION_LENGTH
+                || questionFirstMessage.length() < MIN_QUESTION_LENGTH) {
+            if (questionTitle.length() < MIN_QUESTION_LENGTH || !questionTitle.contains("?")) {
+                return threadChannel.sendMessage(String.format(
+                        """
+                                Your first message or title did not fit the format to be sent to ChatGPT. \
+                                Your message needs to be between %s and %s characters. If your question is longer than %s \
+                                then your title needs to include a question mark."
+                                    """,
+                        MIN_QUESTION_LENGTH, MAX_QUESTION_LENGTH, MAX_QUESTION_LENGTH));
+            }
+            stringBuilder.replace(0, stringBuilder.length(), questionTitle);
+        }
+
+        for (ForumTag tag : tags) {
+            stringBuilder.insert(0, String.format("%s ", tag.getName()));
+        }
+
+        String question = stringBuilder.toString();
+        String response = """
+                Here is an AI assisted attempt to answer your question, maybe it helps! \
+                It will also the tags you selected to provide more context. \
+                In any case, a human is on the way \uD83D\uDC4D
+                """;
+        Optional<String> chatGPTAnswer = chatGptService.ask(question);
+        MessageCreateAction action = threadChannel.sendMessage(response);
+
+        List<MessageEmbed> embedding = List.of(HelpSystemHelper.embedWith(chatGPTAnswer.orElse("""
+                An attempt was made to answer your question with ChatGPT but \
+                an error has occurred while trying to communicate with ChatGPT.
+                """)));
```

Comment:
i think it would be better to not post anything if it fails. like, not even the intial message.